src/ralph/ssl_certificates/management/commands/update_service_env.py

```diff
@@ -0,0 +1,58 @@
+import sys
+
+from django.core.management.base import BaseCommand
+
+from ralph.assets.models.assets import ServiceEnvironment
+from ralph.dns.dnsaas import dnsaas_client
+from ralph.ssl_certificates.models import SSLCertificate
+
+
+class Command(BaseCommand):
+    help = 'Checks the compliance of services in SSL Certificates'
+
+    def update_from_record(self, results):
+        for value in results:
+            domain = value['content']
+            service_dns = value['service_name']
+            name = None
+            try:
+                name = ServiceEnvironment.objects.get(
+                    service__name=service_dns, environment__name='prod'
+                )
+            except ServiceEnvironment.DoesNotExist:
+                self.stderr.write(
+                    'Service with name {} does not exist'.format(service_dns)
+                )
+            SSLCertificate.objects.filter(
+                domain_ssl=domain).update(service_environment=name)
+
+    def update_from_domains(self, results):
+        for value in results:
+            domain = value['name']
+            service_dns = value['service_name']
+            name = None
+            try:
+                name = ServiceEnvironment.objects.get(
+                    service__name=service_dns, environment__name='prod'
+                )
+            except ServiceEnvironment.DoesNotExist:
+                self.stderr.write(
+                    'Service with name {} does not exist'.format(service_dns)
+                )
+            SSLCertificate.objects.filter(
+                domain_ssl=domain).update(service_environment=name)
+
+    def get_records(self):
+        url = dnsaas_client.build_url('records')
+        return dnsaas_client.get_api_result(url)
+
+    def get_domains(self):
+        url = dnsaas_client.build_url('domains')
+        return dnsaas_client.get_api_result(url)
+
+    def handle(self, *args, **options):
+        sys.setrecursionlimit(1500)
```

Comment:
why?

Reply:
Because I got "RuntimeError: maximum recursion depth exceeded" and in this way i fixed it.

Reply:
but you don't have any recursion calls here - could you try to remove it? If this error occurs, please provide (part of) traceback here

Reply:
[Traceback.txt](https://github.com/allegro/ralph/files/2184562/Traceback.txt)

In attachment.

Reply:
DNSaaS returns too many pages, and creator od DNSaaS module in ralph didn't predict it - probably the recursive calls should be changed to non-recursive - or you can filter by relevant record types (like A or CNAME) and/or set limit on a single page (to 100 or sth like that)

Reply:
OK